src/containers/auth/LoginForm.tsx

```diff
@@ -1,9 +1,25 @@
 import Link from 'next/link';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { rem } from 'polished';
 
 import { styled } from '../../lib/styles/stitches.config';
 
 const LoginForm = () => {
+  const [isMobile, setIsMobile] = useState(false);
+
+  const screenChange = (event: MediaQueryListEvent) => {
+    const { matches } = event;
+    setIsMobile(matches);
+  };
+
+  useEffect(() => {
+    const mobileWidthLarge = 640; // @mobileLarge
```

Comment:
```suggestion
    const mobileLargeWidth = 640; // @mobileLarge
```

`mobileLarge`의 `width`이라는 뜻으로 이렇게 바꾸면 조금 더 자연스러워 보일 것 같습니다.